proposals/0020/README.md

```diff
@@ -0,0 +1,151 @@
+---
+id: 0020
+title: Template Versioning
+status: published
+authors: Aman Parauliya <aman@infracloud.io>
+---
+
+## Summary
+
+As per the proposal [0013], we are planning to induce workflow service as a part of the template which is used to create a workflow. As per the future perspective we may again have the requirement of modifying the existing format of template and in that case we should also provide the support of previous/current format of the template in parallel to the modification of template format/schema.  
+This is to provide backward compatibility once [0013] is implemented.
```

Comment:
I think it should be done before we implement 0013.

Reply:
Yes, Actually , let's make it this way that this needs to be done in parallel to 0013 or as a part of 0013.